lib/screens/settings/interactor/viewmodels/settings_state.dart

```diff
@@ -9,28 +9,22 @@ class SettingsState extends Equatable {
   final PageCommand? pageCommand;
   final String? errorMessage;
   final bool hasNotification;
-  final bool? navigateToGuardians;
```

Comment:
Actions are not State, they should be represented by Page Commands. 

State - It holds the UI state, What should the UI draw. 

Navigate to X is the perfect example of an action and not a state. The UI doesnt need to draw anything, it needs to navigate. 

Show dialog is another example of the Action and not a state.